index.js

```diff
@@ -1352,20 +1380,20 @@
   //. //
   //. //   The value at position 1 is not a member of ‘NonZeroInteger’.
   //. ```
-  function NullaryType(name, url, test) {
+  function NullaryType(name, url, parent, test) {
     function format(outer, inner) {
       return outer(stripNamespace(name));
     }
-    return new _Type(NULLARY, name, url, format, test, [], {});
+    return new _Type(NULLARY, name, url, format, parent, test, [], {});
   }
 
   var CheckedNullaryType =
   def('NullaryType',
       {},
-      [String_, String_, Function_([Any, Boolean_]), Type],
+      [String_, String_, Type, Function_([Any, Boolean_]), Type],
       NullaryType);
 
-  //# UnaryType :: String -> String -> (Any -> Boolean) -> (t a -> Array a) -> (Type -> Type)
+  //# UnaryType :: String -> String -> Type -> (Any -> Boolean) -> ((t a -> Array a) -> (Type -> Type))
```

Comment:
To show that we cannot call the function like `('', '', Any, K(true), x => ...)`